app/middlewares/auth.js

```diff
@@ -0,0 +1,26 @@
+const logger = require('../logger');
+const { authError } = require('../errors');
+const { validateToken } = require('../helpers/jwt');
+const {
+  authMessages: { AUTH_ERROR, EXPIRED_TOKEN_ERROR, INVALID_TOKEN_ERROR, VALID_TOKEN }
+} = require('../constants/');
+const {
+  common: {
+    session: { header_name }
+  }
+} = require('../../config');
+
+exports.verifyToken = (req, _, next) => {
+  try {
+    const authHeader = req.headers[header_name];
```

Comment:
You can check if `req.headers[header_name]` exits otherwise you return an authentication error

Reply:
Done!